### ISSUE TITLE
Optionally omit login field when auto-filling

### ISSUE THREAD

**Issue by xeruf** [passff/passff#481]
### General information. Describe your environment

- Versions
  - Operating system: Arch
  - Browser: Firefox
  - PassFF: Version 1.10.6
  - Host app: passff-host 1.2.1

---

### Situation
I have been able to fill in the form at https://secure.erlangen.de/erlangen/form/provide/1465, which I regularly need, remarkably well using this extension. Thank you very much for that :)
```sh
tfAntragstellerName: <Surname>
tfAntragstellerVorname: <Name>
tfAntragstellerBibliotheksausweisnummer: xxxx
tfEMailAntragsteller: ...@pm.me
xi-sel-1_1: 1
tfTelefonnummerlAntragsteller:
tfTitel_Autor1_dyn_tfTitel_Autor1_0:
```
I have thus even been able to autofill the initial radio (`xi-sel-1_1`).

### Issues
But there's two issues here:
- As you can see above, some fields are named by id yet left blank. This is because otherwise passff fills all of them in with the filename of the pass entry, a behaviour I generally dislike,
  ESPECIALLY since it has already overwritten valuable form input previously.
  I think this should be less aggressive.
- I haven't been able to autofill the `select`-dropdown at the bottom and would be grateful for any ideas. My fear is that it cannot be done since it seems changing the selection changes nothing in the HTML.

**Comment by tuxor1337** [comment]
Your point about `<select>` elements is already covered in #390. I just fixed it in cb189f8.

For PassFF aggressively filling in fields with your login name: For your case, I added a feature to omit filling the login field. That means: If there is no login name, you can either add a line `login:` to your pass entry which means that the login name is an empty string. Or you can explicitly instruct PassFF to omit filling any login input fields using the line `login: PASSFF_OMIT_FIELD`.

Does that solve your issue?

**Comment by xeruf** [comment]
That sounds great, thank you!
